### PR TITLE
add models for contact

### PR DESCRIPTION
**Pull request by crackblick**
i have add some Field for the contacts

### CODE REVIEW

File: contacts/models.py [hunk 1]
```diff
@@ -10,14 +10,28 @@ class Meta:
...
+        )
+    SEX = ( 
+        ('MAN', 'Man'),
+        ('WOM', 'Woman'),
+        )
+    type_of_contact = models.CharField(max_length=3, choices=TYPE_OF_CONTACT)
     first_name = models.CharField('First Name', max_length=200)
     last_name = models.CharField('Last Name', max_length=200)
+    birth_date = models.DateField('Birth Date', blank=True)
+    registration_number = models.CharField('National Registration Number', max_length=16, blank=True)
```
Comment (cecton) [inline contacts/models.py]: The full names for this are:
- national identification number
- national identity number
- national insurance number 
https://en.wikipedia.org/wiki/National_identification_number

Maybe it's a bit long for a field name... `id_number` would be short and good enough imho

File: contacts/models.py [hunk 2]
```diff
@@ -10,14 +10,28 @@ class Meta:
...
+    sexe = models.CharField('Sexe', max_length=3, choices=SEX)
```
Comment (cecton) [inline contacts/models.py]: Baguette?

File: contacts/models.py [hunk 3]
```diff
@@ -10,14 +10,28 @@ class Meta:
...
     phone = models.CharField('Phone', max_length=64, blank=True)
+    email = models.EmailField('Email Adress', max_length=254, blank=True)
```
Comment (cecton) [inline contacts/models.py]: Address.

You need to install a spell checker with your source code editor / IDE.

ASAP

File: contacts/models.py [hunk 4]
```diff
@@ -10,14 +10,28 @@ class Meta:
...
 class Contact(models.Model):
+    TYPE_OF_CONTACT = ( 
+        ('CLI', 'Customer'),
+        ('PAT', 'Patient'),
+        ('EMP', 'Employee'),
+        )
+    SEX = ( 
+        ('MAN', 'Man'),
+        ('WOM', 'Woman'),
+        )
```
[the remark this answers is not in the document]
Comment (crackblick) [reply contacts/models.py]: fixe

File: contacts/models.py (hunk 2 above)
Comment (crackblick) [reply contacts/models.py]: xD

File: contacts/models.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (cecton) [reply contacts/models.py]: It's Male/Female

File: contacts/models.py (hunk 1 above)
Comment (cecton) [reply contacts/models.py]: `national_id` might be better

File: contacts/models.py [hunk 5]
```diff
@@ -18,12 +18,12 @@ class Contact(models.Model):
         ('M', 'Man'),
         ('F', 'Female'),
         )
-    type_of_contact = models.CharField(max_length=3, choices=TYPE_OF_CONTACT)
+    type_of_contact = models.CharField(max_length=3, choices=TYPE_OF_CONTACT, blank=False, null=True)
```
Comment (cecton) [inline contacts/models.py]: That doesn't seem to be an optional field

Comment (crackblick) [reply contacts/models.py]: capture d ecran 2019-01-15 a 15 05 26

Comment (cecton) [reply contacts/models.py]: still doesn't explain why it's null=true

File: contacts/models.py [hunk 6]
```diff
@@ -10,14 +9,28 @@ class Meta:
         
 
 class Contact(models.Model):
+    TYPE_OF_CONTACT = ( 
+        ('CLI', 'Customer'),
+        ('PAT', 'Patient'),
+        ('EMP', 'Employee'),
+        )
```
Comment (cecton) [inline contacts/models.py]: The indentation of the last paranthesis is bad. It should be aligned with the word TYPE.

File: contacts/models.py [hunk 7]
```diff
@@ -10,14 +9,28 @@ class Meta:
...
+    GENDER_CHOICES = ( 
+        ('M', 'Male'),
+        ('F', 'Female'),
+        )
```
Comment (cecton) [inline contacts/models.py]: The indentation of the last paranthesis is bad. It should be aligned with the word GENDER.

File: nuke.sh [hunk 8]
```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 
+
```
Comment (cecton) [inline nuke.sh]: This is a typical example of why it is best to use:
```
git add -p
```
And avoid:
```
git add .
```

This is unnecessary in the diff.

**Pull request merged**